contrib/vcloud/vcloudbenchmarkbase.py

```diff
@@ -0,0 +1,128 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.benchexec
+import benchexec.model
+import benchexec.tooladapter
+import benchexec.tools
+import benchexec.util
+
+
+class VcloudBenchmarkBase(benchexec.benchexec.BenchExec):
+    """
+    An extension of BenchExec that supports executing the benchmarks in the
+    VerifierCloud (internal project at https://gitlab.com/sosy-lab/software/verifiercloud/).
+    """
+
+    DEFAULT_OUTPUT_PATH = "results/"
```

Comment:
I still don't understand why this is necessary?

Did you test this and it does not work without?

`benchexec.benchexec.BenchExec` already declares this.

Reply:
sorry, this was my mistake. fixed now